mod-record-specifications-validator/src/main/java/org/folio/rspec/validation/validator/marc/impl/AbstractIndicatorRuleValidator.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.rspec.validation.validator.marc.impl;
+
+import org.folio.rspec.domain.dto.DefinitionType;
+import org.folio.rspec.domain.dto.SeverityType;
+import org.folio.rspec.domain.dto.SpecificationFieldDto;
+import org.folio.rspec.domain.dto.ValidationError;
+import org.folio.rspec.i18n.TranslationProvider;
+import org.folio.rspec.validation.validator.SpecificationRuleValidator;
+import org.folio.rspec.validation.validator.marc.model.MarcField;
+import org.folio.rspec.validation.validator.marc.model.MarcIndicator;
+
+abstract class AbstractIndicatorRuleValidator implements SpecificationRuleValidator<MarcField, SpecificationFieldDto> {
```

Comment:
```suggestion
abstract class AbstractIndicatorRuleValidator implements SpecificationRuleValidator<MarcIndicator, FieldIndicatorDto> {
```
I suggest creating such validator implementation

Reply:
For my opinion it will bring some unnecessary complexity and some code duplication in case of creation of similar MarcRecordRuleValidator with new interface's types MarcIndicator and FieldIndicatorDto.

Reply:
Refactored